python/tests/test_schema.py

```diff
@@ -228,7 +228,7 @@ def test_schema_find_field_by_id_raise_on_unknown_field(table_schema_simple):
 
 
 def test_schema_find_field_type_by_id(table_schema_simple):
-    """Test retrieving a columns's type using its field ID"""
+    """Test retrieving a column type using its field ID"""
```

Comment:
"column's"? I think the possessive form should still be used.

Reply:
Updated!